### PR TITLE
PR fixes for Dynamic Typebindings

### PR DESCRIPTION
**Pull request by Vatyx**

### CODE REVIEW

File: Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 1]
```diff
@@ -1101,12 +1101,12 @@ void USpatialNetDriver::AddActorChannel(const Worker_EntityId& EntityId, USpatia
 
 USpatialActorChannel* USpatialNetDriver::GetActorChannelByEntityId(const Worker_EntityId& EntityId) const
 {
-	USpatialActorChannel* const* ActorChannel = EntityToActorChannel.Find(EntityId);
+	USpatialActorChannel* ActorChannel = EntityToActorChannel.FindRef(EntityId);
```
Comment (m-samiec) [inline Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp]: I meant replace the entire function with `return EntityToActorChannel.FindRef(EntityId);`
I've made this change in my branch actually, so you can just revert these changes.

Comment (improbable-valentyn) [inline Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp]: Mike already merged this. (Also, I believe it just `return EntityToActorChannel.FindRef(EntityId);`)

Comment (Vatyx) [reply Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp]: Yep, he mentioned it
